src/20_Components/amp-experiment.html

```diff
@@ -0,0 +1,80 @@
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+   <!-- #### Setup -->
+  <!--
+    Import the library.
+  -->
+  <script async custom-element="amp-experiment" src="https://cdn.ampproject.org/v0/amp-experiment-0.1.js"></script>
+  <!--
+    Style different variants of the experiment by using `body[amp-x-name-of-experiment="nameOfVariant"`].
+  -->
+  <style amp-custom>
+    body[amp-x-button-color-experiment="yellow"] .test-button{
+      background-color: yellow;
+      color: black;
+    }
+    body[amp-x-button-color-experiment="red"] .test-button{
+      background-color: red;
+    }
+    body[amp-x-button-color-experiment="blue"] .test-button{
+      background-color: blue;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    Configure the experiment realized with `amp-experiment` with a JSON object. Use `variants` within each experiment to declare how many users will be part of an experiment variant. The total sum of the variants must be a number <= 100, if the number is < 100, then the default behaviour is used. An user belongs to one of the variants based on a random number generated between 0 and 100. For more details about variants read the official [documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-experiment/amp-experiment.md). Multiple experiments can be run on the same AMP document in parallel with their own sets of variants. One AMP document can have at most one `amp-experiment` element.
+  -->
+  <amp-experiment>
+    <script type="application/json">
+      {
+        "button-color-experiment": {
+          "variants": {
+            "yellow": 30,
+            "red": 30,
+            "blue": 30
+          }
+        }
+      }
+    </script>
+  </amp-experiment>
+</amp-experiment>
+<!--
+  An experimental button. The background of the button is going to be yellow for the 30% of users, red for 30% of users, blue for 30% of users and, for the remaining 10% of users, the button background is going to be the default color.
+-->
+<button class= "button button-primary test-button" target="_blank">Click here</button>
+
+<!-- #### Reporting -->
+
+<!--
+  Measure results of the experiment using [amp-pixel](https://github.com/ampproject/amphtml/blob/master/builtins/amp-pixel.md). Allocated variants are available as a URL substitution variable: `VARIANT(name-of-the-experiment)`.
+-->
+<amp-pixel src="https://ampbyexample.com/components/amp-experiment?xname=button-color-experiment&xvar=VARIANT(button-color-experiment)"></amp-pixel>
```

Comment:
Add a sentence describing how to view the analytics request in the  network tab of the developer tools.